src/main/webapp/recipe.html

```diff
@@ -250,23 +250,19 @@ <h2>Comments</h2>
               </svg>
             </div>
           </div>
-          <div class="form-group">
-            <textarea class="form-control" id="commentInput" placeholder="Leave a comment..."></textarea>
-          </div>
-          <div class="small-sep">
-            <p>morgansd • San Diego, CA • 6/11/20</p>
-            <p>Absolute knockout. The sour cream folded into the whipped cream is a trick I will 100% steal for other recipes. Perfect amount of acidity throughout the whole thing. Unreal!!</p>
-          </div>
-          <hr class="my-4">
-          <div class="small-sep">
-            <p>emilyc • Seattle, WA • 6/11/20</p>
-            <p>Cakes came out too salty! And a bit dry. The cream tied in the entire dessert so will make again with more cream, less (no) salt, and less time in the oven.
-            </p>
-          </div>
+
+          <form method="POST" action="/new-comment">
+            <input type="text" name="comment-input"/>
+            <br/>
+            <button>Submit</button>
+          </form>
+
```

Comment:
Please remove unnecessary new lines.

Reply:
I'm going to keep the newlines here because it's separating out an extra object. This logically separates the recipe itself from the comment section. I will remove them elsewhere, though.